Dockerfile

```diff
@@ -1,38 +1,64 @@
 FROM ubuntu:18.04
 
-RUN mkdir /data
+RUN mkdir /data /package
 
-ENV BBLFSH_HOSTNAME=tmexp_bblfshd \
-  BBLFSH_PORT=9432 \
-  GITBASE_HOSTNAME=tmexp_gitbase \
-  GITBASE_PORT=3306 \
-  GITBASE_USERNAME=root \
-  GITBASE_PASSWORD=
+ARG DEBIAN_FRONTEND=noninteractive 
+
+ENV BBLFSH_HOSTNAME tmexp_bblfshd
+ENV BBLFSH_PORT 9432
+ENV GITBASE_HOSTNAME tmexp_gitbase
+ENV GITBASE_PORT 3306
+ENV GITBASE_USERNAME root
+ENV GITBASE_PASSWORD ""
+ENV ARTM_SHARED_LIBRARY /usr/local/lib/libartm.so
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN apt-get update \
-  && apt-get install -y dialog apt-utils curl \
+  && apt-get install -y --no-install-recommends \
+  apt-utils \
+  ca-certificates \
+  curl \
+  locales \
+  && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
+  && locale-gen \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 
-RUN curl -sSL https://get.docker.com/ | sh 
-
-RUN apt-get update \
-  && apt-get install -y python3-pip python3-dev \
-  && cd /usr/local/bin \
-  && ln -s /usr/bin/python3 python \
-  && apt-get remove -y .*-doc .*-man >/dev/null \
-  && apt-get autoremove --purge -y \ 
+RUN curl -sSL https://get.docker.com/ | sh \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends \
+  build-essential \
+  cmake \
+  git \
+  libboost-all-dev \
+  make \
+  python3-dev \
+  python3-pip \
+  && ln -s /usr/bin/python3 /usr/local/bin/python \
+  && pip3 install numpy scipy pandas protobuf tqdm wheel \
+  && git clone --branch stable \
+  https://github.com/bigartm/bigartm.git /opt/bigartm \
+  && cd /opt/bigartm \
+  && git reset --hard 3c0cbc4ac08cae37f00754815c105fde930ff379 \
+  && mkdir build \
+  && cd build \
+  && cmake -DINSTALL_PYTHON_PACKAGE=ON -DPYTHON=python3 .. \
+  && make \
+  && make install \
+  && rm -rf /usr/share/doc /usr/share/man \
```

Comment:
after the `make install` int the [ARTM Dockerfile](https://github.com/bigartm/bigartm-docker/blob/master/Dockerfile) they do the following, do we not need to ?
```
RUN cd 3rdparty/protobuf-3.0.0/python && python3 setup.py build && python3 setup.py install
RUN cd python && python3 setup.py install
```

Reply:
I think this one is outdated, last commit is > 2y ago. Here I followed the latest instructions (+ the `-DINSTALL_PYTHON_PACKAGE=ON` that was mentioned nowhereà. Though I tested only that `import artm` works, so please let me know if there is any problem and I will investigate!

Reply:
ah okay, no problem then if something is up we will see :)